packages/appeals/constants/documents.js

```diff
@@ -56,7 +56,7 @@ export const VALID_MIME_TYPES = Object.freeze({
 	},
 	'application/vnd.ms-powerpoint': { hexSignature: 'D0CF11E0', extensions: ['ppt'] },
 	'application/vnd.openxmlformats-officedocument.presentationml.presentation': {
-		hexSignature: 'D0CF11E0',
+		hexSignature: '504B03041400',
```

Comment:
This feels like even more forbidden knowledge than Regex. Is there a reference table for there somewhere online?

Reply:
Google! however this was a bit obvious as all new office docs share the same